config.js

```diff
@@ -72,10 +72,10 @@ const DEFAULT_JOB_QUEUE_TTL_SECONDS_ASYNC = 3600;
 const DEFAULT_JOB_QUEUE_TTL_SECONDS_SYNC = 25;
 
 // default set to 30 minutes
-const DEFAULT_JOB_REMOVAL_DELAY_SECONDS = 1800;
+const DEFAULT_JOB_REMOVAL_INTERVAL_SECONDS = 1800;
 
-const JOB_REMOVAL_DELAY_SECONDS = pe.KUE_JOBS_REMOVAL_DELAY ||
-  DEFAULT_JOB_REMOVAL_DELAY_SECONDS;
+const JOB_REMOVAL_INTERVAL_SECONDS = pe.KUE_JOBS_REMOVAL_INTERVAL ||
```

Comment:
It looks like this variable isn't set in prod or sandbox so there's no action needed.
But this also made me realize there was a variable that was changed in the worker ttl change that is set in prod. (TTL_KUE_JOBS was changed to TTL_KUE_JOBS_ASYNC)

I've updated it in prod and sandbox.